daffodil-tdml-processor/src/main/scala/org/apache/daffodil/tdml/processor/DaffodilTDMLDFDLProcessor.scala

```diff
@@ -350,7 +350,7 @@ class DaffodilTDMLDFDLProcessor private (private var dp: DataProcessor) extends
     val saxOutputChannel = java.nio.channels.Channels.newChannel(saxOutputStream)
     val unparseContentHandler = dp.newContentHandlerInstance(saxOutputChannel)
     unparseContentHandler.enableInputterResolutionOfRelativeInfosetBlobURIs()
-    val xmlReader = SAXParserFactory.newInstance.newSAXParser.getXMLReader
+    val xmlReader = DaffodilSAXParserFactory().newSAXParser.getXMLReader
```

Comment:
This is still calling setSecureDfeaults below. Is that still needed?

Reply:
No, I missed taking that one out.